lonestar/mining/cpu/frequent-subgraph-mining/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+app_mining_cpu(fsm frequent-subgraph-mining)
+
+add_test_mine(small1 frequent-subgraph-mining -symmetricGraph -simpleGraph "${BASEINPUT}/Mining/citeseer.sadj" "-ft=adj")
```

Comment:
This was commented out before due to https://github.com/IntelligentSoftwareSystems/Galois/issues/143. Please leave this commented out until I fix that so we can just get these fixes through more quickly.

Reply:
Oh  I see. I removed comments of some mining tests. I am going to revert them back.